x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/exporter/Exporter.java

```diff
@@ -113,11 +113,11 @@ public void close() {
 
     protected abstract void doClose();
 
-    protected static DateTimeFormatter dateTimeFormatter(final Config config) {
+    protected static DateFormatter dateTimeFormatter(final Config config) {
         Setting<String> setting = INDEX_NAME_TIME_FORMAT_SETTING.getConcreteSettingForNamespace(config.name);
         String format = setting.exists(config.settings()) ? setting.get(config.settings()) : INDEX_FORMAT;
         try {
-            return DateTimeFormat.forPattern(format).withZoneUTC();
+            return DateFormatter.forPattern(format).withZone(ZoneOffset.UTC);
```

Comment:
this also requires a change in 6.7, otherwise users using outdated formats will not be warned if they do so, as this uses the joda time API directly